package.json

```diff
@@ -21,5 +23,21 @@
     "react-dom": "18.2.0",
     "tailwindcss": "3.3.2",
     "typescript": "5.1.6"
+  },
+  "devDependencies": {
+    "@jest/globals": "^29.6.1",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^14.0.0",
+    "@testing-library/react-hooks": "^8.0.1",
```

Comment:
Please, remove this package with npm uninstall because 1) it does not support react 18+ version and break overall project installation. 2) they are shipped automatically with `"@testing-library/react": "^14.0.0",`
![image](https://github.com/Alex-2kZharkov/oldboy-barbershop/assets/66321831/a0ae84f1-2762-4ad0-a8cb-4250675cc57b)